tabbycat/privateurls/views.py

```diff
@@ -99,15 +105,15 @@ def get_adjudicators_table(self):
 
         return table
 
-    def get_tables(self):
+    def get_tables(self) -> List[TabbycatTableBuilder]:
         return [self.get_adjudicators_table(), self.get_speakers_table()]
 
 
 class GenerateRandomisedUrlsView(AdministratorMixin, TournamentMixin, PostOnlyRedirectView):
 
     tournament_redirect_pattern_name = 'privateurls-list'
 
-    def post(self, request, *args, **kwargs):
+    def post(self, request: Any, *args, **kwargs) -> 'HttpResponseRedirect':
```

Comment:
The hint on `request` conceivably be set to WSGIRequest but given that it goes unused in the function body it doesn't matter that much.

Reply:
A similar thing could be set for most of the usages of bare `Any` hints, they could be set to more specific hints but they tend to go unused anyways.